controls/osquery.rb

```diff
@@ -0,0 +1,17 @@
+# encoding: utf-8
+
+title 'Facebook Osquery'
+
+control 'osquery-1' do
```

Comment:
os query is not a requirement in production. While I understand this is a handy tool, I think this should not belong into a core. InSpec allows you to easily write your overlays where you can add you individual production setup on top of this DevSec baseline. See section Profile Inheritance in https://www.inspec.io/docs/reference/profiles/